.github/workflows/tests.yml

```diff
@@ -186,17 +186,39 @@ jobs:
           brew test-bot
         fi
 
-    - name: Deploy the latest Docker image
+    - name: Deploy the latest Docker image to GitHub
       if: matrix.os == 'ubuntu-latest' && github.ref == 'refs/heads/master'
       run: |
         docker login docker.pkg.github.com -u BrewTestBot -p ${{secrets.GITHUB_TOKEN}}
         docker tag brew docker.pkg.github.com/homebrew/brew/brew
         docker push docker.pkg.github.com/homebrew/brew/brew
 
-    - name: Deploy the tagged Docker image
+    - name: Deploy the tagged Docker image to GitHub
       if: matrix.os == 'ubuntu-latest' && startsWith(github.ref, 'refs/tags/')
       run: |
         docker login docker.pkg.github.com -u BrewTestBot -p ${{secrets.GITHUB_TOKEN}}
         v=${GITHUB_REF:10}
         docker tag brew "docker.pkg.github.com/homebrew/brew/brew:$v"
         docker push "docker.pkg.github.com/homebrew/brew/brew:$v"
+
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: ["18.04", "20.04"]
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+      with:
+        fetch-depth: 0
```

Comment:
This might not be needed but I'll defer to you here.

Reply:
It's to unshallow the `Homebrew/brew` repo, so that it doesn't unshallow the first time that `brew update` is run, which adds a big delay to being able to use a fresh container.

Reply:
Gotcha 👍